media-api/app/lib/ImageResponse.scala

```diff
@@ -24,7 +24,16 @@ object ImageResponse {
 
   def fileMetaDataUri(id: String) = URI.create(s"${Config.rootUri}/images/$id/fileMetadata")
 
-  def create(id: String, esSource: JsValue, withWritePermission: Boolean, included: List[String] = List()): (JsValue, List[Link]) = {
+  // TODO: move this as a method of Image (fiddly due to dep on Config)
+  def imageIsPersisted(image: Image) = {
```

Comment:
Can you think of any better place? I'd be keen to move this to the model if possible, but it's a bit of a yakshave. :-(